Darwin/StitchCore/StitchCore/Core/Auth/Internal/StitchAuthImpl.swift

```diff
@@ -333,4 +333,10 @@ internal final class StitchAuthImpl: CoreStitchAuth<StitchUserImpl>, StitchAuth
             }
         }
     }
+
+    func refreshCustomData(_ completionHandler: @escaping (StitchResult<Void>) -> Void) {
```

Comment:
👍 so this is public?

Reply:
Correct, this is exposed publicly through the protocol it's implementing.